### DISCUSSION TITLE
Upgrade Website Template

### DISCUSSION THREAD

**Discussion by ThePlator** [Sanskriti1102/Techtalksintelligenz#20] [General]
![Screenshot 2024-10-05 074838](https://github.com/user-attachments/assets/d583583c-4a3f-473d-8df5-50a2e9f36f71)
1. Will we have different user roles (e.g., admins, contributors, readers)? How will permissions differ?
2. What types of educational content will be hosted?
3. How should we categorize resources? Should users be able to filter or search based on topics?
4. What should be searchable? Only blogs and events, or the entire resource hub as well?
@Sanskriti1102

**Comment by Sanskriti1102** [comment]
@ThePlator  in all the questions, common answer is :
the linkedin description of our Tech Talks IntelligenZ : https://www.linkedin.com/company/104787080/admin/dashboard/
and our youtube : https://www.youtube.com/@TechTalksIntelligenZ2024

In resources : we have medium blogs, Linkedin articles which we intend to feature it on the website which would attract people and according to the theme given in the readme.md

**Comment by Sanskriti1102** [comment]
#7 This is a level 3 issue and is availiable, those contributors who want to contribute to this issue do comment down
